tests/zero_code_change/tf_utils.py

```diff
@@ -232,6 +231,8 @@ def neural_net(x):
 
 
 def get_data() -> "tf.contrib.learn.python.learn.datasets.base.Datasets":
+    from tensorflow.examples.tutorials.mnist import input_data
+
```

Comment:
Why?

Reply:
its causing an error with TF 2 because I'm importing tf_utils in test_estimator.py.

Reply:
Is there a function in this file that is exclusively used with TF 2?
Then we can simply move it to a new utils file, so that we do not have this import clash.

Reply:
there's nothing specific to TF 2. whatever is used in the TF 2 works for both TF 1 and TF 2. only this get data function is specific to TF 1